app/controllers/subject_viewer.coffee

```diff
@@ -53,6 +54,17 @@ class SubjectViewer extends Controller
     @el.attr tabindex: 0
     @setClassification @classification
 
+  setClassificationCohort: (cohort) =>
+    @classification.cohort = cohort
```

Comment:
I don't think this is saved anywhere, you probably want to set `@classification.metadata.cohort`.

See https://github.com/zooniverse/Serengeti/blob/master/app/models/classification.coffee#L37-L48